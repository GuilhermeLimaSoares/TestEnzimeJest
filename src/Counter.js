import React, { Component } from 'react';

class Counter extends Component {
    render() {
        <div className="Counter">
            <h2>VocÃª tem {this.props.count} counter.</h2>

        </div>
    }
}

export default Counter;