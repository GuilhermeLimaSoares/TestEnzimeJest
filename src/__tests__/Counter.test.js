import React from 'react';
import { shallow } from 'enzyme';
import Counter from './../Counter';

describe('Testing Counter', () => {
    it('should render correctly', () => {
        const wrapper = shallow(
            // <Counter count={5} />
            <Counter />
        );

        expect(wrapper).toMatchSnapshot();
        // wrapper.setProps({ count: 10 });
     /*   expect(wrapper).toMatchSnapshot(); */

    });
});