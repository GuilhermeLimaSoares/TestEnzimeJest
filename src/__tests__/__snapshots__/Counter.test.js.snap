// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Counter should render correctly 1`] = `ShallowWrapper {}`;
